app/pages/adduser/confirmation/controller.js

```diff
@@ -1,8 +1,9 @@
 import manifest from './manifest.json';
 import { addParamsToManifest } from '../../../common/helpers/addParamsToManifest';
 
-export const getAddUserConfirmationContext = () => ({
+export const getAddUserConfirmationContext = ({ params, organisationId }) => ({
```

Comment:
why the params? can't userAdded just be passed in instead

Reply:
because it is passed to the `addParamsToManifest` so it made sense

Reply:
incase we add more 🤷‍♀

Reply:
I get it now. I still think this should be passed though as userAdded here and we don't need organisationId.